DataCollector/DoctrineDataCollector.php

```diff
@@ -26,9 +26,16 @@ class DoctrineDataCollector extends BaseCollector
     /** @var string[] */
     private $groupedQueries;
 
-    public function __construct(ManagerRegistry $registry)
+    /** @var bool */
+    private $shouldValidateSchemaClass;
+
+    /**
+     * @param bool $shouldValidateSchemaClass
```

Comment:
please use a typehint instead, as the master branch requires PHP 7.1+

Reply:
done (I had 1.12.x in ming but @ostrolucky confirmed it's master)